docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
<p>You could add labels to the axes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>In this case the sequences are randomly generated so they don't really mean anything, any suggestions on how to label them?</p><p> </p>

Reply:
<p>I think it's still okay to say "Time" on the x-axis and "Random Value" on the y-axis</p>

Reply:
Got it, it's fixed now